src/lib.rs

```diff
@@ -1,4 +1,54 @@
 //! Winit is a cross-platform window creation and event loop management library.
+//! 
+//! # Installation
+//! 
+//! `winit` can be added to `Cargo.toml` as a dependency. It can be added via `cargo add`.
+//! 
+//! ```bash
+//! $ cargo add winit
+//! ```
+//! 
+//! To only enable the X11 backend on Free Unix[^unix] systems, disable default features
+//! and enable the `x11` feature.
+//! 
+//! ```bash
+//! $ cargo add winit --no-default-features --features x11
+//! ```
+//! 
+//! To only enable the Wayland backend on Free Unix systems, disable default features
+//! and enable the `wayland` feature.
+//! 
+//! ```bash
+//! $ cargo add winit --no-default-features --features wayland
+//! ```
+//! 
+//! These features have no effect on systems that are not Free Unix.
+//! 
+//! ## Dependencies
+//! 
+//! Dependencies on non-system libraries is managed through Cargo. For the X11
+//! backend, the following Ubuntu packages or their equivalents must be installed.
+//! 
+//! - `libx11-dev`
+//! - `libxcb1-dev`
+//! - `libxi-dev`
+//! - `libxcbcommon-dev`
+//! - `libxcbcommon-x11-dev`
+//! 
+//! For the Wayland backend, the following Ubuntu packages or their equivalents
+//! must be installed.
+//! 
+//! - `libwayland-dev`
+//! - `libxcbcommon-dev`
+//! 
+//! The "dev" packages are only needed for building binaries that use `winit`. On
+//! deployed system the non-`dev` equivalents need to be installed.
+//! 
+//! The other backends (Windows, macOS, etc) do not have any dependencies on system libraries
+//! that don't already come with the operating system. However, note that the Windows backend
+//! only supports Windows 10 and above. 
```

Comment:
@madsmtm @MarijnS95 I know we support a specific Android version and up, but I'm not clear on macOS.

Reply:
See [`winit::platform::macos`](https://docs.rs/winit/0.30.5/winit/platform/macos/index.html) ;)

Reply:
Indicated as such

Reply:
The latest `master` [shows a different version though](https://rust-windowing.github.io/winit/winit/platform/macos/index.html), so now the two are out of sync :/